<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>336</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Заборов А.М. 224-322 Обучение нейронной сети с помощью генетических алгоритмов</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="input_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>280</width>
      <height>280</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="learn_speed_label">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>135</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Количество данных
для обучения</string>
    </property>
   </widget>
   <widget class="QLabel" name="epoch_count_label">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Количество поколений</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="gens_count_spin_box">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>40</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>999</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_button">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>100</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="learn_button">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Обучить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="identify_button">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>60</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Распознать</string>
    </property>
   </widget>
   <widget class="QLabel" name="status_label">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>280</y>
      <width>281</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSpinBox" name="individs_count_spin_box">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>100</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>10</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="epoch_count_label_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>70</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Количество индивидуумов</string>
    </property>
   </widget>
   <widget class="QLabel" name="epoch_count_label_3">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>180</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Вероятность мутации</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="mutation_prob_spin_box">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>200</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <double>1.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.001000000000000</double>
    </property>
    <property name="value">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QSpinBox" name="data_count_spin_box">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>170</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>10</number>
    </property>
    <property name="maximum">
     <number>12665</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="crossover_prob_spin_box">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>150</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <double>1.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.001000000000000</double>
    </property>
    <property name="value">
     <double>0.500000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="epoch_count_label_4">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>130</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Вероятность скрещивания</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
